etc/workflows/publish.xml

```diff
@@ -181,21 +63,16 @@
     <!-- Clean up work artifacts -->
 
     <operation
-        id="cleanup"
-        fail-on-error="false"
-        description="Remove temporary processing artifacts">
+      id="cleanup"
+      fail-on-error="false"
+      description="Remove temporary processing artifacts">
       <configurations>
+        <!-- On systems with shared workspace or working file repository -->
+        <!-- you want to set this option to false. -->
         <configuration key="delete-external">true</configuration>
-        <!-- FixMe Don't clean up ACLs until workflow service no longer looks for them in the WFR. -->
         <configuration key="preserve-flavors">security/*</configuration>
```

Comment:
Do we still need this? Removing the comment suggest that the problem is fixed? In that case, we should  do a full cleanup.

Reply:
Seems like the ACLs are still required for parsing ActiveMQ messages, so we have to keep them.